### ISSUE TITLE
[Bug]: ERROR: Unable to extract uploader id from yt-dlp

### ISSUE THREAD

**Issue by bbilly1** [tubearchivist/tubearchivist#433]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

any

## Describe the bug
YT has been changing things, breaking yt-dlp and therefore also Tube Archivist. The amazing yt-dlp team is on the case: https://github.com/yt-dlp/yt-dlp/issues/6247, they will publish a release very soon (TM). After that, look out for a release here too, same, also very soon (TM). 

## Steps To Reproduce
Extract any video metadata, download any video.

### Relevant log output

```shell
ERROR: [youtube] Unable to extract uploader id; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by burgsth** [comment]
Seems to be a new know yt-dlp issue which has been fixed in daily build.

https://github.com/yt-dlp/yt-dlp/issues/6253

**Comment by bbilly1** [comment]
New yt-dlp version 2023-02-17 is now shipped in latest release v0.3.3, please update.
